LinuxTraceLogCapture.md

```diff
@@ -124,52 +119,21 @@ $ sudo /usr/bin/perf record -g -a -F 999 -e cpu-clock,sched:sched_stat_sleep,sch
 $ Ctrl-C
 ```
 
-## Convert trace to text format
-This is to useful along-side the CTF trace to resolve UM IP/Symbols. Similar to what [perfcollect](https://raw.githubusercontent.com/microsoft/perfview/master/src/perfcollect/perfcollect) uses
-
-```bash
-$ sudo perf inject -v -s -i perf_cpu.data -o perf.data.merged
-
-# There is a breaking change where the capitalization of the -f parameter changed.
-$ sudo perf script -i perf.data.merged -F comm,pid,tid,cpu,time,period,event,ip,sym,dso,trace > perf.data.txt
-
-if [ $? -ne 0 ]
-then
-    $ sudo perf script -i perf.data.merged -f comm,pid,tid,cpu,time,period,event,ip,sym,dso,trace > perf.data.txt
-fi
-
-# If the dump file is zero length, try to collect without the period field, which was added recently.
-if [ ! -s perf.data.txt ]
-then
-    $ sudo perf script -i perf.data.merged -f comm,pid,tid,cpu,time,event,ip,sym,dso,trace > perf.data.txt
-fi
-```
-
-## Capture trace timestamp start 
-Perf.data.txt only contains relative timestamps. If you want correct absolute timestamps in UI then you will need to know the trace start time.
-
-```bash
-$ sudo perf report --header-only -i perf_cpu.data | grep "captured on"
-```
-
-Place the "captured on" timestamp for example "Thu Oct 17 15:37:36 2019" in a timestamp.txt file next to the trace folder. The timestamp will be interpreted as UTC
-
 # Transferring the files to Windows UI (optional)
 You then need to transfer the perf files to a Windows box where WPA runs. The most important file is perf.data.txt
 
 ```bash
-$ sudo chmod 777 -R perf*
```

Comment:
Just leave this sections alone and add a new section of tracepoint support of perf.data ?